eureka2-core/src/main/java/com/netflix/eureka2/channel/AbstractServiceChannel.java

```diff
@@ -76,6 +76,8 @@ protected boolean moveToState(STATE from, STATE to) {
                     metrics.incrementStateCounter(to);
```

Comment:
With this line replacing all the metrics.incrementStateCount(Idle) lines in the concrete classes' constructors and the metric less moveToState(STATE to) option, can we possibly lose metrics when channels are created and are then immediately closed via a call to moveToState(Closed)?